test/api.js

```diff
@@ -1829,12 +1831,15 @@ describe('api', function() {
       info = sass.info;
 
     it('should return a correct version info', function(done) {
-      assert(info.indexOf(package.version) > 0);
-      assert(info.indexOf('(Wrapper)') > 0);
+      var lines = info.split(eol);
+      
+      assert.equal(lines[0], ['node-sass', package.version, '(Wrapper)', '[JavaScript]'].join('\t'));
+
+      assert(info.indexOf('[C/C++]') > 0);
       assert(info.indexOf('[JavaScript]') > 0);
       assert(info.indexOf('[NA]') < 0);
       assert(info.indexOf('(Sass Compiler)') > 0);
-      assert(info.indexOf('[C/C++]') > 0);
+      assert(info.indexOf('(Sass Language)') > 0);
```

Comment:
I know this is the same as the others, but maybe we can fix up the message parameter at the same time. The current failure just says `AssertionError: false == true`, which doesn't really tell us anything :wink: